docs/examples/ResultsReader.rst

```diff
@@ -0,0 +1,760 @@
+
+Copyright (c) 2017-2018 `Serpent-Tools developer
+team <https://github.com/CORE-GATECH-GROUP/serpent-tools/graphs/contributor>`__,
+GTRC
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS
+OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY
+CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+Results Reader
+==============
+
+Basic operations
+----------------
+
+This notebook demonstrates the capabilities of the serpentTools to read
+Serpent results files. SERPENT [1] produces a result file (i.e.
+``_res.m``), containing general results (e.g. ``k-eff``), metadata (e.g.
+``title``) and homogenized cross-sections. The homogenized cross-section
+sets are printed in the results file for all the requested universes.
+The ResultsReader is capable of reading this file, and storing the data
+inside univ objects. Each such object has methods and attributes that
+should ease the analyses.
+
+ 
+
```

Comment:
I should clarify the developer guide more. Only the `ipython3` phrase needs to be removed. The `.. code::` has to remain or else this the blocks will not be rendered as code at all. This one's on me, and the dev guide will be updated accordingly